src/main/java/net/nikr/eve/jeveasset/gui/shared/filter/FilterManager.java

```diff
@@ -116,12 +125,28 @@ protected void export(List<String> list) {
 				builder.append("] [");
 				builder.append(filter.getLogic().name());
 				builder.append("] [");
-				builder.append(filter.getColumn().name());
+				EnumTableColumn<?> column = filter.getColumn();
+				builder.append(column.name());
 				builder.append("] [");
 				builder.append(filter.getCompareType().name());
 				builder.append("] [");
 				builder.append(wrap(filter.getText()));
-				builder.append("]\r\n");
+				builder.append("]");
+				if (column instanceof FormulaColumn) {
+					FormulaColumn<?> formulaColumn = (FormulaColumn) column;
+					builder.append("[");
+					builder.append(FORMULA);
+					builder.append(formulaColumn.getFormula().getOriginalExpression().replace(":", "::"));
+					builder.append("]");
+				}
+				if (column instanceof JumpColumn) {
+					JumpColumn<?> jumpColumn = (JumpColumn) column;
+					builder.append("[");
+					builder.append(JUMP);
+					builder.append(jumpColumn.getJump().getSystemID());
+					builder.append("]");
+				}
+				builder.append("\r\n");
```

Comment:
Should we add a separator between the prefixes (JUMP/FORMULA) and the data?

Reply:
The reason I ended up not adding one, was that I wasn't sure what one to use, and also, didn't want to used any chars that may be part of the formula, if that makes sense. I'm not sure if that is valid concerns or not.

Reply:
Yeah, I get what you're saying. Also could we potentially have [ or ] in the formula? If so would that break our reading of the filter?

Reply:
Yes, I think that will break it.
It should use the `wrap` function to escape those and then `unwrap` when importing.